lib/modules/manager/custom/regex/readme.md

```diff
@@ -1,5 +1,7 @@
 With the `regex` manager you can configure Renovate so it finds dependencies that are not detected by its other built-in package managers.
 
+Renovate uses the `ECMAScript (JavaScript)` flavor of regex.
+
```

Comment:
I think this can be removed 🤔

@rarkins WDYT?

Reply:
I put this info at the top of the page, so that more users will see it. The other mention of the flavor is "hidden" in the `### Regular Expression Capture Groups` wall of text.

I'll let `@rarkins` decide what to do. 😄

Reply:
Is it fully true through? I thought we kind of use the re2 flavor of regex.

It would definitely be ideal if we can help people who want to use a tool like regex101

Reply:
ok, re2 uses ecmascript regex flavor too. it simply doesn't support the backtracking and lookahead/ look behind stuff.

so the regex needs to be a valid ecmascript regex first to be parsable

Reply:
we should also link to the re2 limitations here 

https://github.com/uhop/node-re2?tab=readme-ov-file#limitations-things-re2-does-not-support

Reply:
Maybe we can just mention that we use `re2` and add link along with it... writing ECMA script floavour of regex might just confuse users a bit till they realize we use `re2`.

Reply:
```suggestion
Renovate uses the `ECMAScript (JavaScript)` flavor of regex.

Renovate uses the `uhop/node-re2` package that provides bindings for [`google/re2`](https://github.com/google/re2).
Read about [`uhop/node-re2`'s limitations in their readme](https://github.com/uhop/node-re2#limitations-things-re2-does-not-support).
```

How about something like this?

Reply:
Minor modification:

```suggestion
Renovate supports the `ECMAScript (JavaScript)` flavor of regex.
```

Reply:
sounds good to me